src/data/polynomial/ring_division.lean

```diff
@@ -499,7 +516,7 @@ begin
 end
 
 lemma count_map_roots [is_domain A] {p : A[X]} {f : A →+* B} (hf : function.injective f)
```

Comment:
I see that you didn't change this, but why is `p` implicit here?

Reply:
It was used only once, as in `simpa ... using count_map_roots hf` so making `p` implicit saves two characters ... I've now made `p` explicit, though the lemma is not even used now.

Reply:
Oh, the use is only removed in a WIP branch ...